### PR TITLE
[DO NOT MERGE - WIP only] Update composer dependencies 2018-07-24

### PR DESCRIPTION
**Pull request by phil-davis**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -316,170 +248,33 @@
...
-                "eventmanager",
-                "persistence",
-                "spl"
-            ],
-            "time": "2017-08-31T08:43:38+00:00"
-        },
         {
             "name": "doctrine/dbal",
-            "version": "v2.7.1",
+            "version": "v2.8.0",
```
Comment (DeepDiver1975) [inline composer.lock]: @phil-davis this is all too much ... we have a PR open on dbal update and it fails.
Please let the bot do it's job.
This way we are wasting too much valuable development time

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

File: composer.lock (hunk 1 above)
Comment (phil-davis) [reply composer.lock]: Actually here I am just trying to sort out fixing the webUI  tests issue that happens every time the bot tries to bump something, and "accidentally" gets the ``MinkSelenium2Driver`` bump that has a regression.

I do not want to merge this PR as it is.

Comment (phil-davis) [reply composer.lock]: Once I have sorted the ``MinkSelenium2Driver`` issues, I will make a PR for just what is needed.

**Comment by phil-davis** [comment]
See #32144 for the proposed real fix/workaround.

**Pull request closed without merging**